vars/cfManifestSubstituteVariables.groovy

```diff
@@ -0,0 +1,125 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.variablesubstitution.ExecutionContext
+import com.sap.piper.variablesubstitution.DebugHelper
+import com.sap.piper.variablesubstitution.YamlUtils
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS + [
+    /**
+     * The `String` path of the Yaml file to replace variables in.
+     * Defaults to "manifest.yml" if not specified otherwise.
+     */
+    'manifestFile',
+    /**
+     * The `String` path of the Yaml file to produce as output.
+     * If not specified this will default to `manifestFile` and overwrite it.
+     */
+    'outputManifestFile',
+    /**
+     * The `String` path of the Yaml file containing the variables' values to use as a replacement in the manifest file.
+     * Defaults to `manifest-variables.yml` if not specified otherwise.
+     */
+    'variablesFile'
+]
+
+@Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS
+
+/**
+ * Step that substitutes variables in a given YAML file with those specified in a another. The format to reference a variable
+ * in the YAML file is to use double parentheses `((` and `))`, e.g. `((variableName))`.
+ * A declaration of a variable and assignment of its value is simply done as a property in the variables YAML file.
+ *
+ * The format follows <a href="https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#variable-substitution">Cloud Foundry standards</a>.
+ *
+ * The step is activated by the presence of both a `manifest.yml` and a variables file. Names of both files are configurable.
+ *
+ * Usage: `cfManifestSubstituteVariables manifestFile: "path/to/manifest.yml", variablesFile:"path/to/manifest-variables.yml", script: this`
+ *
+ * @param arguments - the map of arguments.
```

Comment:
This appears as strange erratic artifact in the docu. I would like to propose to remove this.

Reply:
Hi there! What exactly should I remove? The entire documentation on class level?
Or just the usage? Since I cannot really reproduce what this looks like, can you please clarify what should be removed?

Reply:
<img width="729" alt="Screen Shot 2019-08-26 at 11 08 38" src="https://user-images.githubusercontent.com/1853183/63679619-0156f580-c7f2-11e9-876f-0d533606cdf5.png">


i'm talking here about the line `@param arguments`